tests/jballerina-unit-test/src/test/resources/test-src/expressions/access/member_access.bal

```diff
@@ -1178,6 +1178,64 @@ function testUnspecifiedFieldRecordMemberAccessWithStringCharKeyExpr() {
     assertTrue(v is ());
 }
 
+type INT int;
+type INT2 int|int:Signed16;
+type STR2 string|string:Char;
```

Comment:
for names how about `Int1`, `Int2`, `Str1`?

Reply:
Changed to `IntType1` etc, is that acceptable?

Reply:
Ok. Then, IntType1, IntType2, StrType1 ?

Reply:
Currently it's `IntType`, `IntType2`, `Str`, will change as originally suggested for a common format